clients/admin-ui/src/features/dataset/EditDrawer.tsx

```diff
@@ -24,6 +25,10 @@ interface Props {
   deleteTitle: string;
   deleteMessage: ReactNode;
   children: ReactNode;
+  /* Associates the submit button with a form, which is useful for when the button
+   * does not live directly inside the form hierarchy
+   */
+  formId?: string;
```

Comment:
Tip: block comments matching
```js
/**
 * ...
 */
```
will be picked up by JS documentation tools better, like VS code intellisense:
<img width="598" alt="Screen Shot 2022-11-17 at 12 33 46" src="https://user-images.githubusercontent.com/2236777/202554196-02707971-70e2-4cc2-a3e8-fdb2f70c3a35.png">

Reply:
oh that's a great tip, thanks!